packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -0,0 +1,123 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/booter-lb3app
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {BootBindings, Booter} from '@loopback/boot';
+import {CoreBindings, inject} from '@loopback/core';
+import {
+  ExpressRequestHandler,
+  OpenApiSpec,
+  rebaseOpenApiSpec,
+  RestApplication,
+} from '@loopback/rest';
+import * as debugFactory from 'debug';
+import {Application as ExpressApplication} from 'express';
+import pEvent from 'p-event';
+import * as path from 'path';
+
+const {generateSwaggerSpec} = require('loopback-swagger');
+const swagger2openapi = require('swagger2openapi');
+
+const debug = debugFactory('loopback:boot:lb3app');
+
+const DefaultOptions: Lb3AppBooterOptions = {
+  // from "/dist/application.ts" to "/legacy"
+  path: '../legacy/server/server',
```

Comment:
Should it be `'../lb3app/server/server'` now?

Reply:
I thought you were referring to changing it only for the tests, but I'll update it here too then.

Reply:
On second thought, throughout the file the LB3 app is referred to as `legacyApp`, do you think we should still change the folder to `lb3app`?

Reply:
I think `lb3app` reflects the intension more accurately. People interprets `legacy` differently.

Reply:
If you really want to be consistent, feel free to rename `legacyApp` to `lb3App`.